cost/google/unattached_volumes/README.md

```diff
@@ -4,6 +4,7 @@
 
 This Policy Template scans all volumes in the given account and identifies any unattached volumes that have been unattached for at least the number of user-specified days. If any are found, an incident report will show the volumes, and related information and an email will be sent to the user-specified email address.
 
+If APIs & services are not enabled for a project, the policy will skip that particular project and executes it. On the next run if services are enabled, then the project will be taken into consideration.
```

Comment:
I don't understand the part **and execute it**.  It executes what.  I thought it is skipped.  Can you clarify and make this more clear.  Maybe I'm not understanding.  thx

Reply:
Fixed It. Please let me know your thoughts.